src/localization/fr-FR.json

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
T-h-i-s- -i-s- -s-o- -c-o-o-l-!- -T-h-a-n-k-s- -f-o-r- -i-n-t-r-o-d-u-c-i-n-g- -t-h-i-s- -t-o- -u-s-!

Reply:
G-l-a-d- -y-o-u- -l-i-k-e-d- -i-t-! 😀

I've coded this quickly, but ideally fake translation should introduce characters from different languages so that it tests that our font support them all.

Also for the actual demos to customers we might want to replace this with the real translations (even from Google Translate). I can imagine how this can confuse business people or give them a feeling that the store is somehow broken.